### ISSUE TITLE
Suggestion: disable for this file

### ISSUE THREAD

**Issue by ciaoben** [Microsoft/vscode-spell-check#27]
It would be nice to be able to disable the plugin for the current opened file. Or at least disable the plug in itself. 
I has to uninstall it because I was writing an article in italian, and everything was red 😃

**Comment by seanmcbreen** [comment]
:)

This is a great suggestion - I could add in a new section to the config file e.g.

IgnoreFilesList : ["somefilename.tex","anothername.md"]

Would something like that work?

Sean

**Comment by Spown** [comment]
it would be great if specific files (![glob](https://github.com/isaacs/node-glob)) could be parsed with different languages, ie config:
```js
"language" : {
  "en": ["**/*-en.js", ".locales/en-US/*.json"],
  "de": ["**/*-de.js", ".locales/DE-DE/*.json"],
  "none": ["./lib/**/*.js"] //will be ignored
}
```
This would enable multilang support as well

**Comment by seanmcbreen** [comment]
This is related to #22 so I'll close this one but link them.  I agree this would be a nice feature.  It did not make it into the most recent update but is very doable.

**Comment by seanmcbreen** [comment]
I just resolved #22 with a toggle action in the status bar hope it helps and if you like it LMK in the marketplace rating.
